### ISSUE TITLE
Custom TMBarIndicator: Overriding non-open instance method outside of its defining module

### ISSUE THREAD

**Issue by PawelDecowski** [uias/Tabman#367]
### New Issue Checklist

- [✓] Updated Tabman to the latest version
- [✓] I took a look at the [Troubleshooting Guide](https://github.com/uias/Tabman/blob/master/Docs/TROUBLESHOOTING.md)
- [✓] I searched for [existing GitHub issues](https://github.com/uias/Tabman/issues)

### Issue Description

Trying to create a custom indicator.

According to the [docs](https://uias.github.io/Tabman/master/going-custom.html#custom-bar-indicator), I need to subclass `TMBarIndicator` and override its `layout` method:

```swift
class CustomBarIndicator: TMBarIndicator {
    override func layout(in view: UIView) {
        super.layout(in: view)

        // Create your indicator in `view`.
    }
}
```

But the `layout` method isn’t declared `open` so it can’t be overridden from outside of its module:

```Overriding non-open instance method outside of its defining module```

**Comment by msaps** [comment]
@PawelDecowski oops, fix is on the way! 🎉

**Comment by msaps** [comment]
Fixed in [2.1.3](https://github.com/uias/Tabman/releases/tag/2.1.3) 🎉
